theme_anelusia/views/snippets/s_features_wall.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_features_wall" inherit_id="website.s_features_wall">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="data-oe-shape-data">{"shape":"web_editor/Floats/09"}</attribute>
+    </xpath>
+    <!-- Shape -->
+    <xpath expr="//div[hasclass('o_we_shape')]" position="replace">
+        <div class="o_we_shape o_web_editor_Floats_09 o_we_animated"/>
+    </xpath>
```

Comment:
nitpicking (?) but I'd replace the attributes rather than rewrite the whole element

Feel free to tell me what you think about it 👍 

If you agree with the suggestion, don't forget to adapt other occurrences.

```suggestion
    <xpath expr="//div[hasclass('o_we_shape')]" position="attributes">
        <attribute name="class" remove="o_web_editor_Airy_04_001" add="o_web_editor_Floats_09" separator=" "/>
        <!-- Remove the background image -->
        <attribute name="style"/>
    </xpath>
```

Reply:
@anso-odoo this is definitly nitpicking, you can just ignore this one 😅